cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -53,7 +53,7 @@ def test_retire_service(appliance, setup_provider, context, provision_request):
 
 @pytest.mark.meta(blockers=[BZ(1544535, forced_streams=['5.9'])])
 @pytest.mark.parametrize('context', [ViaSSUI])
-@pytest.mark.parametrize('provision_request', [['console_test']], indirect=True)
+@pytest.mark.parametrize('order_service', [['console_test']], indirect=True)
```

Comment:
please use provision_request fixture here as it will wait for request to finish else immediately after provisioning the service , your test will start .
Also , please add  "assert provision_request.is_succeeded()" in the provision request fixture so that your test begins only when the request is successful.

Reply:
are you sure that is how we can make it wait until `provision_request` is successful? The problem with parameterizing `provision_request` is that I can not access the `request parameters` in `order_service` fixture and hence in `order_service` `if hasattr(request, 'param'):` yields false and console template is not selected.

Reply:
Okay so ..
https://github.com/kedark3/integration_tests/blob/ee39ca813897f3e03ff397afb644bc57867b7b6a/cfme/fixtures/service_fixtures.py#L87 
will wait for the request to finish.

I see that you cannot access the request param with provision_request, so we can combine both fixtures in one method ..

@pytest.fixture
def order_service(appliance, provider, provisioning, vm_name,
                  dialog, catalog, request):
    """ Orders service once the catalog item is created"""
    if hasattr(request, 'param'):
        catalog_item = create_catalog_item(provider, provisioning, vm_name, dialog, catalog,
                                    console_test=True if 'console_test' in request.param else None)
    else:
        catalog_item = create_catalog_item(provider, provisioning, vm_name, dialog, catalog)
    service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog,
                                       catalog_item.name)
    service_catalogs.order()
    provision_request = appliance.collections.requests.instantiate(catalog_item.name,
                                                                   partial_check=True)
    provision_request.wait_for_request(method='ui')
    assert provision_request.is_succeeded()
    return catalog_item
# TODO - remove request finally